packages/nextjs/components/Footer.tsx

```diff
@@ -12,21 +14,32 @@ import { getTargetNetwork } from "~~/utils/scaffold-eth";
  */
 export const Footer = () => {
   const nativeCurrencyPrice = useGlobalState(state => state.nativeCurrencyPrice);
+  const isLocalNetwork = getTargetNetwork().id === hardhat.id;
```

Comment:
I think we should probably move this util to `network.ts`, since we are using it in a bunch of places.

We could create and issue + another PR.

Reply:
Yup agree ! Will create a PR for it 🙌